src/components/LocaleSelector.vue

```diff
@@ -0,0 +1,79 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2023 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="locale-selector">
+    <select
+      v-model="$i18n.locale"
+      :aria-label="$t('select-language')"
+    >
+      <option
+        v-for="(lang, i) in langs"
+        :key="`lang-${i}`"
+        :value="lang"
+      >
+        {{ getLanguageName(lang) }}
```

Comment:
I'm not sure this needs to be extracted into a function since we're just accessing an object like `locales[lang]`

Reply:
this is something I fixed later on in this PR: https://github.com/marinaaisa/swift-docc-render/pull/8/